infrastructure/src/main/java/org/corfudb/infrastructure/CompactorService.java

```diff
@@ -127,8 +127,12 @@ private void runOrchestrator() {
                 managerStatus = (CheckpointingStatus) txn.getRecord(
                         CompactorMetadataTables.COMPACTION_MANAGER_TABLE_NAME,
                         CompactorMetadataTables.COMPACTION_MANAGER_KEY).getPayload();
+                if (managerStatus == null && isLeader) {
+                    txn.putRecord(getCompactorLeaderServices().getCompactorMetadataTables().getCompactionManagerTable(),
+                            CompactorMetadataTables.COMPACTION_MANAGER_KEY,
+                            CheckpointingStatus.newBuilder().setStatus(StatusType.IDLE).setEpoch(0).build(), null);
```

Comment:
Here the epoch is set to 0, but the compactor's runtime epoch might be something else. What's the effect of this discrepancy?

Reply:
But this is only on the first run right? 
Or the case when we don't have the status. That means we have lost the status of previous runs. The checkpointers also check the epoch before every table's checkpoint. If there's a mismatch they stop.

Reply:
This epoch is different from the runtime's epoch. This epoch denotes the compactor cycle number. It's incremented at every initCompactionCycle().
https://github.com/CorfuDB/CorfuDB/blob/master/infrastructure/src/main/java/org/corfudb/infrastructure/CompactorLeaderServices.java#L88

I realize the naming might be misleading

Reply:
Renamed epoch to cycle_count